src/lint/lint.ts

```diff
@@ -87,3 +104,64 @@ export async function lint(options: Options, config: ProjectConfig) {
     return new CommandResult(1);
   }
 }
+
+async function fix(
+    warnings: FixableWarning[],
+    options: Options,
+    config: ProjectConfig,
+    analyzer: Analyzer,
+    analysis: Analysis) {
+  if (options.input) {
+    const packageRelativeInputFilenames = new Set(options.input.map(
+        (i) => path.relative(config.root, path.resolve('.', i))));
+    warnings = warnings.filter(
+        (w) => packageRelativeInputFilenames.has(w.sourceRange.file));
+  }
+
+  const fixes = warnings.map((w) => w.fix).filter((fix) => fix !== undefined) as
+      Replacement[][];
+  if (fixes.length === 0) {
```

Comment:
Consider adding: When I run this CLI command my intention is to lint for errors and then fix any that are found. With that understanding, it would be more clear to log "No errors found" if there were no warnings to begin with.